emily/handler/src/bin/emily-lambda.rs

```diff
@@ -0,0 +1,37 @@
+//! Emily API entrypoint.
+
+use emily_handler::context::EmilyContext;
+use tracing::info;
+use warp::Filter;
+
+use emily_handler::api;
+use emily_handler::logging;
```

Comment:
I think this is unused. Weird, I cannot get clippy or vs code to point it out.

Reply:
Changed logging import to use this.